setup.py

```diff
@@ -1,15 +1,20 @@
 from __future__ import absolute_import
 from setuptools import setup
 
+
 setup(
     name="chem",
-    version="0.3.0",
```

Comment:
Since you're unpinning scipy and numpy, this could potentially break upstream packages that might be accidentally relying on this behavior.  For other repos where we do this sort of thing, we've been going with a major version number bump instead of a minor version update.

Reply:
@feanil Wait, do you mean _pinning_ (not unpinning)?

I'd probably prefer to have these be unpinned in the library, `openedx-chem`,
and pinned in the application, `edx-platform`.

That's what we did in `openedx-calc` last time:
- unpinned dependencies
- bumped minor version

I pushed that change up (to unpin);
I can't for the life of me remember why I did it that way.
In my defense, I wrote the first iteration of this initiative 4+ years ago ;)

Reply:
Ah sorry, not enough words on my part.  What I meant to say is, unpinning is correct, we should do it. But it might break up-stream so we should do a major version bump instead of a minor one.  I would update this to 1.0.0 and then I think this is good to go.

Reply:
@stvstnfrd does this make sense?  it looks like we've unpinned which I think is correct.  Now we just need to bump to 1.0.0 and this is good to merge.